src/index.js

```diff
@@ -24,6 +26,8 @@ if (__DEV__) {
 // TODO: Empty object until we get persistant storage hooked up
 const store = CreateStore({});
```

Comment:
Is this going to cause issues without having persistent storage, where if an app closes or loses mount, any data may be tossed if not stored in AsyncStorage and synced? Just a thought. I see the auth handled there, which looks grood!